src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class CalcUI extends React.Component {
+  render() {
+    return (
+      <div className="divCalculator">
+        <div className="divButtons">
+          <div className="divResult">0</div>
+          <button className="btnNumber" type="button">AC</button>
```

Comment:
[OPTIONAL]

- [ ] You can create a single Button component that can be reused throughout your application which is one of the beauty of react 😄